channels/nz.m3u

```diff
@@ -1,27 +1,33 @@
 https://bcsecurelivehls-i.akamaihd.net//hls/live/720612/1614493167001_1/master_iPad1080p.m3u8
-#EXTINF:-1 tvg-id="MaoriTV.nz" tvg-name="Maori TV" tvg-country="NZ" tvg-language="" tvg-logo="" group-title="",Maori TV
+#EXTINF:-1 tvg-id="MaoriTV.nz" tvg-name="Maori TV" tvg-country="NZ" tvg-language="Maori" tvg-logo="https://freeviewnz.tv/nonumbracoimages/ChannelsOpg/freeview-channel-logos-38.png" group-title="",Maori TV
 https://bcsecurelivehls-i.akamaihd.net/hls/live/720612/1614493167001_1/master_iPad720p.m3u8
 #EXTINF:-1 tvg-id="tv.13" tvg-name="Māori Television" tvg-country="NZ;AU" tvg-language="Maori" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/freeview-channel-logos-38.png" group-title="",Māori Television (1080p)
 https://bcsecurelivehls-i.akamaihd.net//hls/live/720612/1614493167001_1/master.m3u8
 #EXTINF:-1 tvg-id="tv.13" tvg-name="Māori Television" tvg-country="NZ;AU" tvg-language="Maori" tvg-logo="http://i.mjh.nz/nz/images/tv.13.png" group-title="",Māori Television (1080p)
 https://i.mjh.nz/nz/tv.13.m3u8
-#EXTINF:-1 tvg-id="tv.18" tvg-name="Parliament TV" tvg-country="NZ" tvg-language="English" tvg-logo="http://i.mjh.nz/nz/images/tv.18.png" group-title="Legislative",Parliament TV (1080p)
+#EXTINF:-1 tvg-id="parliament.nz" tvg-name="Parliament TV" tvg-country="NZ" tvg-language="English" tvg-logo="https://i.imgur.com/BsHt9f2.png" group-title="Legislative",Parliament TV (1080p)
 https://i.mjh.nz/nz/tv.18.m3u8
+#EXTINF:-1 tvg-id="parliament.nz" tvg-name="Parliament TV" tvg-country="NZ" tvg-language="English" tvg-logo="https://i.imgur.com/BsHt9f2.png" group-title="Legislative",Parliament TV (1080p)
+https://ptvlive.kordia.net.nz/out/v1/daf20b9a9ec5449dadd734e50ce52b74/index.m3u8
 #EXTINF:-1 tvg-id="Rialto.nz" tvg-name="Rialto" tvg-country="NZ;AU" tvg-language="English" tvg-logo="https://i.imgur.com/RnzWT9C.jpg" group-title="",Rialto (1080p)
 https://rialto-rialto-samsungaustralia.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="" tvg-logo="http://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/te-reo.png" group-title="",Te Reo
+#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="Maori" tvg-logo="http://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/te-reo.png" group-title="",Te Reo
 https://bcsecurelivehls-i.akamaihd.net//hls/live/720613/1614493167001_2/master_iPad1080p.m3u8
-#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="" tvg-logo="" group-title="",Te Reo
+#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="" tvg-logo="http://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/te-reo.png" group-title="",Te Reo
 https://bcsecurelivehls-i.akamaihd.net/hls/live/720613/1614493167001_2/master_iPad720p.m3u8
```

Comment:
same case here, master link is  https://bcsecurelivehls-i.akamaihd.net/hls/live/720613/1614493167001_2/master.m3u8